### ISSUE TITLE
Using tensorboardX vs torch.utils.tensorboard

### ISSUE THREAD

**Issue by neevparikh** [alex-petrenko/sample-factory#46]
Is there a reason tensorboardX is used instead of the supported default https://pytorch.org/docs/stable/tensorboard.html?

**Comment by alex-petrenko** [comment]
No particular reason, I guess when the development started this wasn't available, or it's just a legacy from the previous version of the repo.
Any advantage from switching to torch.utils.tensorboard (except one less dependency)?

**Comment by neevparikh** [comment]
Nope just that. Probably not worth the switch, considering that
tensorboardX is still actively developed.

Best,
Neev Parikh
Scb. Computer Science

**Comment by neevparikh** [comment]
I was actually wondering if there's an advantage to using that and was
curious, haha.

Best,
Neev Parikh
Scb. Computer Science

**Comment by alex-petrenko** [comment]
I'll remember to use this in my next project LOL.

Honestly, Tensorboard is not a very good tool. I'm excited to try Weights and Biases at some point.
